test/metric_value_benchmark/container_insights_test.go

```diff
@@ -54,8 +54,8 @@ func (t *ContainerInsightsTestRunner) GetAgentConfigFileName() string {
 	return "./agent_configs/container_insights.json"
 }
 
-func (t *ContainerInsightsTestRunner) GetAgentRunDuration() time.Duration {
-	return test_runner.MinimumAgentRuntime
+func (t *ContainerInsightsTestRunner) getAgentRunDuration() time.Duration {
```

Comment:
Because I reduced agent runtime for ec2 tests. Let ecs stay as 1 minute run time.